tests/functional/adapter/materialized_view_tests/fixtures.py

```diff
@@ -30,7 +30,6 @@ def query_relation_type(project, relation: BaseRelation) -> Optional[str]:
 materialized_view = """
 {{ config(
     materialized='materialized_view',
-    description='this is a materialized view',
```

Comment:
Removed as this is not a valid way to specify a comment, and thus was not even addressed in the tests.